docs/sources/v0.50.x/set-up/set-up-distributed-k6/upgrade-k6-operator.md

```diff
@@ -0,0 +1,8 @@
+---
+weight: 200
+title: Upgrade k6-operator
+---
+
+# Upgrade k6-operator
+
+<!-- TODO: Add content -->
```

Comment:
The issue for writing these instructions is here:
https://github.com/grafana/k6-operator/issues/371

Should we hide this section for now?

Reply:
That makes sense, I added the frontmatter property to hide this for now. :)